google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -62,6 +62,10 @@ def insert_violations(self, violations, resource_name,
             # Create the violations snapshot table.
             snapshot_table = self._create_snapshot_table(
                 resource_name, snapshot_timestamp)
+        except MySQLdb.OperationalError, e:
```

Comment:
not sure I understand what's happening here. If the table is already there, why try to create it again?